builtin/providers/aws/resource_aws_autoscaling_group.go

```diff
@@ -414,6 +407,11 @@ func getAwsAutoscalingGroup(
 func resourceAwsAutoscalingGroupDrain(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).autoscalingconn
 
+	if d.Get("force_delete").(bool) {
+		log.Printf("[DEBUG] Skipping ASG drain, force_delete was set.")
```

Comment:
Not sure what you mean - it's `true` if we get to line 411. Is it just the log wording you'd like changed?